src/Microsoft.Sbom.Targets/SbomCLIToolTask.cs

```diff
@@ -29,7 +29,41 @@ protected override string GenerateFullPathToTool()
     /// <returns>string list of args</returns>
     protected override string GenerateCommandLineCommands()
     {
-        return "Command";
+        var arguments =
+                "generate " +
+                $"-BuildDropPath {this.BuildDropPath} " +
+                $"-BuildComponentPath {this.BuildComponentPath} " +
+                $"-PackageName {this.PackageName} " +
+                $"-PackageVersion {this.PackageVersion} " +
+                $"-PackageSupplier {this.PackageSupplier} " +
+                $"-NamespaceUriBase {this.NamespaceBaseUri} " +
+                $"-DeleteManifestDirIfPresent {this.DeleteManifestDirIfPresent} " +
+                $"-FetchLicenseInformation {this.FetchLicenseInformation} " +
+                $"-EnablePackageMetadataParsing {this.EnablePackageMetadataParsing} " +
+                $"-Verbosity {this.Verbosity} ";
```

Comment:
Enclose the paths in quotes. Something like ` $"-BuildDropPath \"{this.BuildDropPath}\" "`

Reply:
Updated! The CommandLineBuilder automatically appends quotes for the arguments